Vic2ToHoI4/Source/Mappers/Graphics/GraphicsMapperFactory.cpp

```diff
@@ -68,6 +69,23 @@ void Mappers::GraphicsMapper::Factory::loadLeaderPortraitMappings(const std::str
 	}
 }
 
+void Mappers::GraphicsMapper::Factory::loadFemalePortraitMappings(const std::string& cultureGroup,
+	 const std::map<std::string, std::vector<std::string>>& portraitMappings)
+{
+	auto cultureGroupMappings = graphicsMapper->femalePortraitMappings.find(cultureGroup);
+	if (cultureGroupMappings == graphicsMapper->femalePortraitMappings.end())
```

Comment:
```suggestion
	if (auto cultureGroupMappings = graphicsMapper->femalePortraitMappings.find(cultureGroup); cultureGroupMappings == graphicsMapper->femalePortraitMappings.end())
```

Reply:
Cannot be done as we still need cultureGroupMappings in the following for loop.